packages/frint-router-react/src/Link.spec.js

```diff
@@ -0,0 +1,264 @@
+/* eslint-disable import/no-extraneous-dependencies, func-names, no-unused-expressions */
+/* global describe, it, before, resetDOM */
+import { expect } from 'chai';
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import { MemoryRouterService } from 'frint-router';
+
+import Link from './Link';
+
+function createContextWithRouter(router) {
+  return {
+    app: {
+      get(key) {
+        if (key === 'router') {
+          return router;
+        }
+
+        return null;
+      }
+    }
+  };
+}
+
+describe('frint-route-react › Link', () => {
+  before(function () {
+    resetDOM();
+  });
+
+  it('renders anchor with href, className and children when no type prop is passed', () => {
```

Comment:
unless needed, please consider using `function () { }` everywhere, just like how it's done in `before`.